src/Workspaces/Core/Portable/Rename/RenameRewriterParameters.cs

```diff
@@ -25,7 +26,7 @@ internal class RenameRewriterParameters
         internal readonly ISymbol RenameSymbol;
         internal readonly string ReplacementText;
         internal readonly bool ReplacementTextValid;
-        internal readonly ISet<TextSpan> StringAndCommentTextSpans;
+        internal readonly ImmutableDictionary<TextSpan, ImmutableSortedSet<TextSpan>> StringAndCommentTextSpans;
```

Comment:
should we have a dedicated type for this?

Reply:
That seems a bit too heavy, especially it is only used at one place in the rewriter.